extractphpfiles.py

```diff
@@ -0,0 +1,11 @@
+import os, glob
+
+folder = "wordpress/"
+ext = "*.php"
+for filename in [y for x in os.walk(folder) for y in glob.glob(os.path.join(x[0], ext))]:
```

Comment:
Is part of this a very complex way of saying `os.path.splitext(...)[1] == "php"`?

Reply:
I think this used to recursively traverse `folder` and find all `php` files in it. But it seems the recursive-ness has since been removed, so I can probably rewrite this.